portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,53 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+ function getShows() {
+  const showsContainer = document.getElementById('shows-container');
+  removeChildren(showsContainer);
+  const imgs = ["images/IMG-9491.jpg", "images/IMG-9492.jpg", "images/IMG-9493.jpg",
+                "images/IMG-9494.jpg"];
+  const office = "A show I can watch over and over and it's hilarious every time";
+  const bb = "Breaking Bad: Crazy storytelling, will have you on the edge of your seat the whole way through";
+  const op = "An adventure story that is hilarious and inspiring";
+  const btc = "Prequel to Breaking Bad, really well written and acted.  Can't wait for season 6.";
+  const pars = [office, bb, op, btc];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random image which features quotes from my favorite TV shows and movies.
+  const index = Math.floor(Math.random() * imgs.length);
+  const src = imgs[index];
+  const p = pars[index];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  let image = document.createElement("img");
```

Comment:
The `<template>` construct is a much less error prone way to build a structured document fragment of HTML from JavaScript and insert it into the webpage. Consider trying it out.